crates/teloxide/src/dispatching/dispatcher.rs

```diff
@@ -295,15 +295,39 @@ where
     /// [`shutdown`]: ShutdownToken::shutdown
     pub async fn dispatch_with_listener<'a, UListener, Eh>(
         &'a mut self,
-        mut update_listener: UListener,
+        update_listener: UListener,
         update_listener_error_handler: Arc<Eh>,
     ) where
         UListener: UpdateListener + 'a,
         Eh: ErrorHandler<UListener::Err> + 'a,
         UListener::Err: Debug,
+    {
+        self.try_dispatch_with_listener(update_listener, update_listener_error_handler)
+            .await
+            .expect("try_dispatch_with_listener failed")
+    }
+
+    /// Same as `dispatch_with_listener` but returns a Result<_> instead of
+    /// panicking
```

Comment:
```suggestion
    /// Same as `dispatch_with_listener` but returns a `Err(_)` instead of
    /// panicking when the initial `get_me` call fails.
```